scripts/buildlsp.sh

```diff
@@ -1,18 +1,30 @@
 #!/bin/sh
+#
+# Builds the copilot lsp, then copies the generated dist/ folder to
+# marimo/_lsp/copilot
+
+# This script should fail if any intermediate step fails
+set -e
 
 cd lsp
 if pnpm build; then
   echo "Removing old lsp files..."
   rm -rf ../marimo/_lsp/
   echo "Copying new lsp files..."
-  mkdir ../marimo/_lsp
+  mkdir -p ../marimo/_lsp
```

Comment:
`set -e` is awesome idea. But why is this broken? Given `rm -rf ../marimo/_lsp/`? If this was a local issue, there's a
chance that some file with bad permissions could not be deleted